debug_toolbar/settings.py

```diff
@@ -42,6 +43,7 @@
     "SQL_WARNING_THRESHOLD": 500,  # milliseconds
     "OBSERVE_REQUEST_CALLBACK": "debug_toolbar.toolbar.observe_request",
     "TOOLBAR_LANGUAGE": None,
+    "DEBUG_TOOLBAR_CONFIG": "test" in sys.argv,
```

Comment:
I don't think we should use `DEBUG_TOOLBAR_CONFIG as the name for this setting. It should be related to the fact that the application is running the test suite. Some other options are:

- TESTING
- RUNNING_TESTS
- IS_TESTING
- TEST

Reply:
I think I will change it to `RUNNING_TESTS`